bikeshed/include/html/header.include

```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  <title>[TITLE] [LEVEL]</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="w3c-status" content="[STATUS]">
-  <link href="../default.css" rel=stylesheet type="text/css">
-  <link href="../csslogo.ico" rel="shortcut icon" type="image/x-icon">
-  <link href="styles/styles-html.css" rel=stylesheet type="text/css">
-  <link href="[W3C-STYLESHEET-URL]" rel=stylesheet type="text/css">
+  <title>[TITLE] [LEVEL]</title>
```

Comment:
I'm still *super* confused.  What, precisely, are you trying to do?  The `Title` and `H1` metadata's *sole purpose* is to serve as appropriate things to fill the `<title>` and `<h1>` elements.  Why are you trying to set them to something different and then correct it in the template?